### PR TITLE
Feature/20160608 fixes

### PR DESCRIPTION
**Pull request by noahmanger**
- Fixes an issue where the sticky menu on the advanced data page didn't stick
- Adjusts the title of the advanced data page and replaces hard-coded title with a template variable

### CODE REVIEW

File: openfecwebapp/templates/advanced.html [hunk 1]
```diff
@@ -2,18 +2,18 @@
...
+  {{ title }}
 {% endblock %}
 
 {% block body %}
-  {{ header.header('Advanced data', '', show_search=False) }}
+  {{ header.header(title, '', show_search=False) }}
   <div class="main">
   <div class="container">
-    <h1 class="icon-heading--circle-table">Advanced data index</h1>
+    <h1 class="icon-heading--circle-table">{{ title }}</h1>
```
Comment (adborden) [inline openfecwebapp/templates/advanced.html]: Not a big deal, but this does change the copy: Advanced data index -> Advanced data

Comment (noahmanger) [inline openfecwebapp/templates/advanced.html]: Yep, that was the intention. @emileighoutlaw noted the desired change [here](https://github.com/18F/openFEC-web-app/issues/1245#issuecomment-223981578)

**Comment by adborden** [comment]
Not a huge deal, but this branch looks like it's based on `develop`. If the fix is targeted for the release, it should be based on the release branch with something like `git checkout -b bug/release-fix origin/release/release-20160608`. That will avoid pulling in extra commits happening on `develop`.

**Comment by adborden** [comment]
Oh, I should add, that of course you would need to create an extra PR to merge into `develop` if you want the fix to also exist on dev right away. Otherwise, the fix will eventually land on `develop` once the `release` branch is closed via git flow.

**Comment by noahmanger** [comment]
Oh thanks. Yeah, I should have branched off of release. And yeah, it's ok to wait for this.

**Pull request merged**